src/build/__main__.py

```diff
@@ -8,6 +8,7 @@
 
 from typing import List, Optional, TextIO, Type, Union
 
+import build as _build
```

Comment:
Can you explain why we need this weird alias?

Reply:
We already define `build` in the file.

Reply:
Perhaps then we shouldn't if we clash with our package name.